plugins/woocommerce-docs/composer.json

```diff
@@ -4,15 +4,17 @@
 	"type": "wordpress-plugin",
 	"require": {
 		"woocommerce/action-scheduler": "^3.6",
-		"erusev/parsedown": "^1.7"
+		"league/commonmark": "^2.4"
```

Comment:
Originally I moved to commonmark because its much more extensible. While I don't need the extensibility now we are running 2 passes I think having the extensibility could come up in future so I'm sticking with it for now.